### PR TITLE
build: disable pseudolocales in Electron

### PR DESCRIPTION
**Pull request by codebytere**
#### Description of Change

Remove pseudocode locales from Electron, which were added in https://chromium-review.googlesource.com/c/chromium/src/+/2774898 and rolled in https://github.com/electron/electron/pull/28462. We don't use these.  

See https://electronhq.slack.com/archives/CC8J5QC00/p1618961550011900?thread_ts=1618932845.005500&cid=CC8J5QC00.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

File: shell/browser/native_window_mac.mm [hunk 1]
```diff
@@ -330,8 +330,8 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
     styleMask |= NSWindowStyleMaskClosable;
   if (resizable_)
     styleMask |= NSResizableWindowMask;
-  if (!useStandardWindow || transparent() || !has_frame())
```
Comment (MarshallOfSound) [inline shell/browser/native_window_mac.mm:333]: Oops?

**Review by MarshallOfSound — APPROVED** [review]

File: shell/browser/native_window_mac.mm (hunk 1 above)
Comment (codebytere) [reply shell/browser/native_window_mac.mm]: already removed lol

File: build/args/all.gn [hunk 2]
```diff
@@ -22,4 +22,8 @@ dawn_enable_vulkan_validation_layers = false
 # This breaks native node modules
 libcxx_abi_unstable = false
 
+# Electron does not use these.
```
Comment (nornagon) [inline build/args/all.gn:25]:
```suggestion
# These are disabled because they cause the zip manifest to differ between
# testing and release builds.
```

**Review by jkleinsc — APPROVED** [review]

**Pull request merged**